tests/test_job.py

```diff
@@ -287,3 +287,12 @@ async def coro() -> NoReturn:
     await scheduler.spawn(coro())
     await scheduler.close()
     handler.assert_called_once()
+
+
+async def test_job_name_set(scheduler: Scheduler) -> None:
```

Comment:
```suggestion
@pytest.skipif(sys.version_info < (3, 8), reason="API not available on 3.7")
async def test_job_name_set(scheduler: Scheduler) -> None:
```
And skip this as well.